libs/dh/wholesale/feature-start/src/lib/dh-wholesale-start.component.ts

```diff
@@ -14,29 +14,71 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Component, NgModule } from '@angular/core';
+import { ChangeDetectionStrategy, Component, NgModule } from '@angular/core';
 import { TranslocoModule } from '@ngneat/transloco';
-import { WattButtonModule } from '@energinet-datahub/watt';
+import {
+  WattButtonModule,
+  WattDatepickerModule,
+  WattFormFieldModule,
+  WattRangeValidators,
+  WattDropdownModule,
+  WattDropdownOption,
+} from '@energinet-datahub/watt';
+import { FormControl, ReactiveFormsModule } from '@angular/forms';
 
 import { DhWholesaleBatchDataAccessApiStore } from '@energinet-datahub/dh/wholesale/data-access-api';
 import { DhFeatureFlagDirectiveModule } from '@energinet-datahub/dh/shared/feature-flags';
+import { CommonModule } from '@angular/common';
 
 @Component({
   selector: 'dh-wholesale-start',
+  changeDetection: ChangeDetectionStrategy.OnPush,
   templateUrl: './dh-wholesale-start.component.html',
   styleUrls: ['./dh-wholesale-start.component.scss'],
   providers: [DhWholesaleBatchDataAccessApiStore],
 })
 export class DhWholesaleStartComponent {
   constructor(private store: DhWholesaleBatchDataAccessApiStore) {}
 
+  optionsGridAreas: WattDropdownOption[] = Object.keys({
+    805: '805',
+    806: '806',
+  }).map((key) => ({
+    displayValue: key,
+    value: key,
+  }));
+
+  formControlGridArea = new FormControl<string[] | null>(null);
```

Comment:
suggestion: instead of adding the `required` on the `watt-dropdown` component in the HTML template, you can set the FormControl as required. This is recommended when using Reactive driven forms.

```suggestion
  formControlGridArea = new FormControl<string[] | null>(null, Validators.required);
```

Note: `Validators` is imported from `@angular/forms`.

Reply:
I have implemented the suggested change